Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -132,9 +133,10 @@ std::string NANDImporter::GetPath(const NANDFSTEntry& entry, const std::string&
 
 std::string NANDImporter::FormatDebugString(const NANDFSTEntry& entry)
 {
-  return StringFromFormat("%12.12s 0x%02x 0x%02x 0x%04x 0x%04x 0x%08x 0x%04x 0x%04x 0x%04x 0x%08x",
-                          entry.name, entry.mode, entry.attr, entry.sub, entry.sib, entry.size,
-                          entry.x1, entry.uid, entry.gid, entry.x3);
+  return fmt::format(
+      "{:12.12} {:#04x} {:#04x} {:#06x} {:#06x} {:#010x} {:#06x} {:#06x} {:#06x} {:#010x}",
```

Comment:
This change in width seems a bit weird, at least I can't see any indication in the docs that the width used for the alternate form _includes_ the prefix.

Then again, it probably doesn't matter too much since this is a debug string.

Reply:
I've tested this code, and it works correctly if I write it like this.

For what it's worth, https://fmt.dev/latest/syntax.html contains the following example (but other than that I indeed can't find anything relevant in the documentation):
```
fmt::format("{:#04x}", 0);
// Result: "0x00"
```